values/src/main/java/org/spine3/time/LocalDates.java

```diff
@@ -48,15 +52,119 @@ public static LocalDate now() {
+        checkPositive(day, "day");
+        final LocalDate result = LocalDate.newBuilder()
+                                          .setYear(year)
+                                          .setMonth(month)
+                                          .setDay(day)
+                                          .build();
+        return result;
+    }
+
+    /**
+     * Obtains a copy of this local date with the specified number of years added.
+     */
+    public static LocalDate plusYears(LocalDate localDate, int yearsToAdd) {
+        checkPositive(yearsToAdd, "yearsToAdd");
+        return changeYear(localDate, yearsToAdd);
+    }
+
+    /**
+     * Obtains a copy of this local date with the specified number of months added.
      */
-    public static LocalDate plusDays(int daysToAdd) {
+    public static LocalDate plusMonths(LocalDate localDate, int monthsToAdd) {
+        checkPositive(monthsToAdd, "monthsToAdd");
+        return changeMonth(localDate, monthsToAdd);
+    }
+
+    /**
+     * Obtains a copy of this local date with the specified number of days added.
+     */
+    public static LocalDate plusDays(LocalDate localDate, int daysToAdd) {
+        checkPositive(daysToAdd, "daysToAdd");
+        return changeDays(localDate, daysToAdd);
+    }
+
+    /**
+     * Obtains a copy of this local date with the specified number of years subtracted.
+     */
+    public static LocalDate minusYears(LocalDate localDate, int yearsToSubtract) {
+        checkPositive(yearsToSubtract, "yearsToSubtract");
+        return changeYear(localDate, -yearsToSubtract);
+    }
+
+    /**
+     * Obtains a copy of this local date with the specified number of months subtracted.
+     */
+    public static LocalDate minusMonths(LocalDate localDate, int monthsToSubtract) {
+        checkPositive(monthsToSubtract, "monthsToSubtract");
+        return changeMonth(localDate, -monthsToSubtract);
+    }
+
+    /**
+     * Obtains a copy of this local date with the specified number of days subtracted.
+     */
+    public static LocalDate minusDays(LocalDate localDate, int daysToSubtract) {
+        checkPositive(daysToSubtract, "daysToSubtract");
+        return changeDays(localDate, -daysToSubtract);
+    }
+
+    private static LocalDate changeYear(LocalDate localDate, int yearsToChange) {
+        final Calendar calendar = Calendar.getInstance();
+        calendar.set(Calendar.YEAR, localDate.getYear());
+        // The Calendar class assumes JANUARY is zero. Therefore subtract 1 to set the value of Calendar.MONTH.
+        calendar.set(Calendar.MONTH, localDate.getMonthValue() - 1);
+        calendar.set(Calendar.DAY_OF_MONTH, localDate.getDay());
+        calendar.add(Calendar.YEAR, yearsToChange);
+        final int year = calendar.get(Calendar.YEAR);
+        // The Calendar class assumes JANUARY is zero. Therefore add 1 to get the value of MonthOfYear.
+        final MonthOfYear month = MonthOfYear.forNumber(calendar.get(Calendar.MONTH) + 1);
```

Comment:
You repeat this over and over again. Why don't you create a method looking like this:

```java
public static MonthOfYear getMonth(Calendar calendar) {
````